sklearn/metrics/tests/test_classification.py

```diff
@@ -1628,3 +1629,20 @@ def test_brier_score_loss():
     # calculate even if only single class in y_true (#6980)
     assert_almost_equal(brier_score_loss([0], [0.5]), 0.25)
     assert_almost_equal(brier_score_loss([1], [0.5]), 0.25)
+
+
+@if_numpydoc
+def test_docstring():
+    # Test for consistency among docstring of different metrics
+    assert_consistent_docs([precision_recall_fscore_support, precision_score,
+                            recall_score, f1_score, fbeta_score],
+                           include_returns=False,
+                           exclude_params=['labels', 'average', 'beta'])
```

Comment:
why not beta?

Reply:
we want to test average for precision_score, recall_score, f1_score, fbeta_score. Can use a separate assertion, I suppose.

Reply:
We have inconsistency in parameter `beta`
For `precision_recall_fscore_support`,
`beta : float, 1.0 by default
        The strength of recall versus precision in the F-score.`
For `fbeta_score`,
`beta : float
        Weight of precision in harmonic mean.`